lib/logic/triggers.js

```diff
@@ -60,14 +60,12 @@ LineCommentsTrigger.TYPE = "line_comments";
 
 class PullRequestListTrigger extends Trigger {
     static TYPE: string;
-    myPulls: Array<PullRequest>;
-    assignedPulls: Array<PullRequest>;
+    pulls: Object;
     error: ?string;
 
-    constructor(myPulls: Array<PullRequest>, assignedPulls: Array<PullRequest>, error: ?string) {
+    constructor(pulls: Object, error: ?string) {
```

Comment:
See https://review.rocks/#/repos/Kegsay/github-pull-review/73/diffs?_k=fxq05m#cmt-47760898

I would expect that `PullRequestListTrigger` would take a `PullRequest[]` rather than an `Object`, and then bucket them if you need to (I don't think you will need to given you can trivially do `prList.filter(function(pr) { return pr.myStatus === "needs_action"; });` or whatever.